api/core/migration_helpers.py

```diff
@@ -8,9 +8,14 @@
 
 class PostgresOnlyRunSQL(migrations.RunSQL):
     @classmethod
-    def from_sql_file(cls, file_path: str, reverse_sql: str) -> "PostgresOnlyRunSQL":
-        with open(file_path) as f:
-            return cls(f.read(), reverse_sql=reverse_sql)
+    def from_sql_file(
+        cls, file_path: str, reverse_sql: str = None, reverse_sql_file_path: str = None
```

Comment:
We should probably add validation here to ensure that `reverse_sql` and `reverse_sql_file_path` aren't used together? I guess an alternative would be to use a single argument and try to infer if it's a file path (e.g. `os.file.exists()`).